docs/manage/_command_line.rst

```diff
@@ -83,7 +83,7 @@ The path should be to a folder named ``KOLIBRI_DATA`` at the root of the local d
 Create a New Super User
 -----------------------
 
-In case you need to create another Super user, either to address additional need of managing facility, or if you lost the password for the old one, run the following command:
+In case you need to create another Super user, either to address additional need of managing facility, or if you lost the password for the old one, run the following command.
```

Comment:
in Kolibri, we use the term 'superuser', not 'super user'

Reply:
I've been using the term "superuser" in all documentation and it the term also already appears in Kolibri as "superuser". I think the special connotation behind "superuser" might also get lost in various translations. The way I see it is: the user isn't just a user that is "super", they have a special label called "superuser"

Reply:
I agree with @khangmach - it's not a user that is super, it's a noun and a specific role in Kolibri. I vote "superuser."